src/store/ChartAdapterStore.ts

```diff
@@ -192,6 +191,30 @@ export default class ChartAdapterStore {
                     [activeItems[index1], activeItems[index2]] = [activeItems[index2], activeItems[index1]];
                 },
             },
+            drawingTool: {
+                onAdd: () => {
+                    this.mainStore.drawTools.onCreation();
+                },
+                onUpdate: () => {
+                    this.mainStore.drawTools.onUpdate();
+                },
+                onRemove: () => {
+                    // this.mainStore.drawTools.deleteGroupTool(index);
+                },
+                onEdit: () => {
+                    // console.log('From the On Edit', index);
+                    // this.mainStore.studies.editStudyByIndex(index);
+                },
```

Comment:
Let's remove these if they are not needed.

Reply:
Removed now , thanks